calendar.js

```diff
@@ -3,38 +3,35 @@ function myTable() {
     var table = createTable();
     getScheduleAndUpdateTable(table);
     document.body.appendChild(table);
-    // myFunction(table);
-    $("#calendarId").on('click', 'td', myFunction);
-
 }
 myTable();
+$("#calendar-id").on('click', 'tr', myFunction);
 
-function myFunction() {
-    var modal = $('#myModal');
-    modal.css('display', 'block');
-    var span = $('.close');
-    var closebtn = $('.btn.btn-default');
 
-    var closefn = function () {
-        modal.css('display', 'none');
-    }
-    span.click(closefn);
-    closebtn.click(closefn);
-
-    // When the user clicks anywhere outside of the modal, close it
-    window.onclick = function (event) {
-        if (event.target == modal) {
-            modal.css('display', 'none');
+function myFunction() {
+ // var updateButton = document.getElementById('updateDetails');
+  var cancelButton = document.getElementById('cancel');
```

Comment:
you can use jquery shortcuts here as well, please stick to one consistent style - use jquery throughout
`$('#cancel)` and so on.